apps/hash-frontend/src/pages/[shortname].page.tsx

```diff
@@ -96,6 +101,215 @@ const ProfilePage: NextPageWithLayout = () => {
 
   const profileNotFound = !profile && !loading;
 
+  const baseTabs = useMemo<ProfilePageTab[]>(
+    () => [
+      {
+        kind: "profile",
+        title: "Profile",
+      },
+      {
+        kind: "profile-pages",
+        title: "Page",
+        pluralTitle: "Pages",
+      },
```

Comment:
Could we just add the Page's entityTypeId into `pinnedEntityTypeBaseUrls` automatically here and let the functions to handle other pinned types deal with fetching and sorting it into groups?

We only need to special case it to stop people removing it, the rest of the functionality should be the same, no?

Reply:
I can do that, but I'm not really convinced this is worth it. It'd complicate updating `pinnedEntityTypeBaseUrls` in all contexts for something we know we'll always be special casing (when we discussed in standup the other day we concluded we'd want users/orgs to have the non-removable `Pages` tab for the foreseeable future).

It'd also need to be added wherever we create users, so that `pinnedEntityTypeBaseUrls` is populated correctly.

Reply:
Ah I get it, apologies for misunderstanding I should've thought of that. Agree this would simplify things, have addressed in https://github.com/hashintel/hash/pull/3338/commits/207953360b4752b0ede415934f7f88c48fad71f3